pkg/model/api_loadbalancer.go

```diff
@@ -31,14 +31,18 @@ type APILoadBalancerBuilder struct {
 var _ fi.ModelBuilder = &APILoadBalancerBuilder{}
 
 func (b *APILoadBalancerBuilder) Build(c *fi.ModelBuilderContext) error {
-	// Configuration where an ELB fronts the master (apiservers in particular)
-
+	// Configuration where an ELB fronts the API
 	if !b.UseLoadBalancerForAPI() {
 		return nil
 	}
 
+	elbSpec := b.Cluster.Spec.API.ELB
+	if elbSpec == nil {
```

Comment:
Added a comment, and cleaned up the unnecessary second `if`, but I'm trying to avoid changing the whole function